src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -76,8 +86,13 @@ public Command parseCommand(String userInput) throws ParseException {
             return new UnmarkCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
-            return new ListCommandParser().parse(arguments);
-
+            if (!descriptor.equals("")) {
```

Comment:
```suggestion
            if (!descriptor.isEmpty()) {
```
Would it be better to use the isEmpty() method in the java String class to reduce the need for magic literals?